test/burn/org/apache/cassandra/utils/LongBTreeTest.java

```diff
@@ -785,7 +784,7 @@ public void testBatchesSmallOverlappingRange() throws ExecutionException, Interr
     @Test
     public void testIndividualInsertsMediumSparseRange() throws ExecutionException, InterruptedException
     {
-        testInsertions(randomSeed(), perThreadTrees / 10, 500, 10, 1, true);
+        testInsertions(randomSeed(), 500, 10, 1, true);
```

Comment:
the default value of this `tests` parameter is `perThreadTrees * threads`, so we could be replacing values of `1000` with `25600` (where there are 32 availableProcessors). Note in ant we limit this to `2 processors` [here|https://github.com/apache/cassandra/blob/trunk/build.xml#L127] so a value of `1600` (which is still an increase).

is this what you intended @Mmuzaf ?

Reply:
Correct, I think we can safely increase the number of local runs for `testLargeBatchesLargeRange`, and `testIndividualInsertsMediumSparseRange` while still maintaining a reasonable execution time for the LongBTreeTest as a whole. 

For the tests that fire `testInsertions` with parameters the main concern we have is - we perform different numbers of test attempts to check tree insertions for variations of ranges and batches, but the same equal number of `tests` should be used for all of them. I think `perThreadTrees * threads` is the better choice for us since increasing the `perThreadTrees` (for `testInsertions` it's not exactly trees per thread as it says, it's just a multiplier) will give us the ability to find patterns e.g. for a heap consumption during a particular run.

Reply:
It is also possible to just set the `perThreadTrees` for 10000 back to 100 as an _easy fix_, I guess. I just wanted to fix the problem wider and maybe I overdid it :-)